src/apps/replication/lib/mutation.cpp

```diff
@@ -184,7 +184,7 @@ int mutation::clear_log_task()
     return 0;
 }
 
-mutation_ptr mutation_queue::add_work(int code, dsn_message_t request, replica* r)
+mutation_ptr mutation_queue::add_work(int code, dsn_message_t request, replica* r, bool disable_batch)
```

Comment:
instead of adding a new option, we should make the parameter to the mutation queue configurable, so that disable-batch can be achieved by simply one possible configuration.